examples/huggingface/pytorch/text-generation/quantization/run_tuning.sh

```diff
@@ -238,7 +238,7 @@ function run_tuning {
         script="run_generation_sq.py"
     elif [ "${topology}" = "llama2_7b_gptq" ]; then
         model_name_or_path="meta-llama/Llama-2-7b-hf"
-        extra_cmd=$extra_cmd" --woq --bits 4 --compute_dtype fp32 --scheme asym "
+        extra_cmd=$extra_cmd" --woq --bits 4 --compute_dtype fp32 --scheme asym --calib_iters 100"
```

Comment:
why 100, the official setting of gptq and awq  is 128, autoround is 512

Reply:
it means calibration run inference iteration number.
`autoround` use "iters", but it has another meaning that how many benchmark inference iterations to compute the throughput  in itrex, so use "calib_iters", and "calib_iters" also is used by smoothquant before, it is the same meaning to express how many iterations number in calibration function.
`gptq` doesn't has this parameter.